api/mastodon.ts

```diff
@@ -0,0 +1,26 @@
+import got from '../libs/got'
+import { millify } from '../libs/utils'
+import { createBadgenHandler, PathArgs } from '../libs/create-badgen-handler'
+
+export default createBadgenHandler({
+  title: 'Mastodon/Pleroma',
+  examples: {
+    '/mastodon/follow/1': 'followers count (mastodon.social)',
+    '/mastodon/follow/1/mas.to': 'followers count (custom Mastodon instance)',
+    '/mastodon/follow/1/pleroma.site': 'followers count (custom Pleroma instance)',
+  },
+  handlers: {
+    '/mastodon/follow/:userId/:instance?': handler
+  }
+})
+
+async function handler ({ userId, instance = 'mastodon.social' }: PathArgs) {
+  const prefixUrl = `https://${instance}/api/v1/`
+  const info = await got(`accounts/${userId}`, { prefixUrl }).json<any>()
+
+  return {
+    subject: `follow @${info.username}@${instance}`,
```

Comment:
Two `@` looks a bit duplicate, maybe remove the first one?
```suggestion
    subject: `follow ${info.username}@${instance}`,
```

Reply:
I agree but that is a convention in Mastodon world:

<img src="https://user-images.githubusercontent.com/1170440/82596492-aba85180-9ba7-11ea-9f4d-5c6ca7c9f7c6.png" width="320">

https://mastodon.social/@Gargron

Reply:
Yea, let's go with convention 👍